src/NextAudio.Matroska/README.md

```diff
@@ -0,0 +1,52 @@
+# Documentation
```

Comment:
Is this section essential? I've looked at the main README and there's nothing there that is not here.